app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/devmode/DevContentFragment.kt

```diff
@@ -78,7 +80,7 @@ class DevContentFragment : Fragment() {
         binding?.devContentWebView?.apply {
             settings.javaScriptEnabled = true
             setBackgroundColor(Color.TRANSPARENT)
-            if (WebViewFeature.isFeatureSupported(WebViewFeature.FORCE_DARK)) {
+            if (context.isNightMode() == true && WebViewFeature.isFeatureSupported(WebViewFeature.FORCE_DARK)) {
```

Comment:
nit, same here (right? maybe I'm missing something).

Reply:
Same here, refactored:)

The function also returned null when the system was not sure which theme is set, I refactored it now to only return true or false, so also removed the "== true" from this code